catalog/plugin/catalog-plugin-videothumbnail/pom.xml

```diff
@@ -123,7 +123,7 @@
                         <configuration>
                             <rules>
                                 <ArtifactSizeEnforcerRule implementation="org.codice.maven.artifactsize.ArtifactSizeEnforcerRule">
-                                    <maxArtifactSize>26.4_MB</maxArtifactSize>
+                                    <maxArtifactSize>56.4_MB</maxArtifactSize>
```

Comment:
Is this a separate change we need?

Reply:
I didn't know if it needed done separately, I just added it here because it is an easy fix.

Reply:
This looks unrelated to this PR.